entgql/internal/todo/ent/entc.go

```diff
@@ -17,14 +17,15 @@ import (
 
 func main() {
 	ex, err := entgql.NewExtension(
-		entgql.WithWhereFilters(true),
-		entgql.WithSchemaPath("../ent.graphql"),
-		entgql.WithConfigPath("../gqlgen.yml"),
+		entgql.WithConfigPath("./gqlgen.yml"),
+		entgql.WithSchemaGenerator(),
+		entgql.WithSchemaPath("./ent.graphql"),
+		entgql.WithWhereInputs(true),
 	)
 	if err != nil {
 		log.Fatalf("creating entgql extension: %v", err)
 	}
-	err = entc.Generate("./schema", &gen.Config{
+	err = entc.Generate("./ent/schema", &gen.Config{
```

Comment:
Please add a comment explaining the codegen is executed from internal/todo and not internal/todo/ent.